microsoft-365/admin/misc/pilot-microsoft-365-from-my-custom-domain.md

```diff
@@ -137,10 +178,10 @@ When you complete this step, all email sent to usera@yourcompany.com and userb@y
 
     - Verify that forwarding is set up properly from an outside account, or from an employee email account on the existing email system. For example, from the original server account for User C or a Hotmail account, send User A an email and verify that it arrives in the Microsoft 365 mailbox for User A.
 
-### Step 9: Move mailbox contents
+### Step 10: Move mailbox contents
 
 Because you are moving only two test users, and User A and User B are both using Outlook, you can move the email by opening the old .PST file in the new Outlook profile and copying the messages, calendar items, contacts, and so on. For more information, see [Import email, contacts, and calendar from an Outlook .pst file](https://support.microsoft.com/office/import-email-contacts-and-calendar-from-an-outlook-pst-file-431a8e9a-f99f-4d5f-ae48-ded54b3440ac).
 
 After they’re imported to the appropriate locations in the Microsoft 365 mailbox, the items can be accessed from any device, anywhere.
 
-When more mailboxes are involved, or if employees are not using Outlook, you can use the migration tools available in the Exchange admin center. To get started, go to Exchange admin center, and follow the directions in [Migrate Email from an IMAP Server to Exchange Online Mailboxes – we need a new article resource].
\ No newline at end of file
+When more mailboxes are involved, or if employees are not using Outlook, you can use the migration tools available in the Exchange admin center. To get started, go to Exchange admin center, and follow the directions in [Migrate Email from an IMAP Server to Exchange Online Mailboxes – we need a new article resource].
```

Comment:
This paragraph should really not be published as it is. We either need a link to the reference or the last sentence should be deleted.

Reply:
Have linked a reference document. THank you.